recipes/tethys-platform/meta.yaml

```diff
@@ -0,0 +1,118 @@
+{% set name = "tethys-platform" %}
+{% set version = "3.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tethysplatform/tethys/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 3775d5599e756206204e09ca9a2114c8866fad7fd29999b256343c847ada3b1f
+
+build:
+  noarch: python
+  number: 0
+  ignore_run_exports:
+    - pip
+    - pbr
+  entry_points:
+    - tethys = tethys_cli:tethys_command
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - pbr
+  run:
+    - python >=3.6
+    - pycrypto =2.6.*
+    - pyopenssl =22.0.*
+    - docker-py =5.0.*
+    - distro =1.6.*
+    - postgresql
+    - psycopg2 <2.9
+    - sqlalchemy =1.4.*
+    - geoalchemy2 =0.6.*
+    - plotly =5.9.*
+    - bokeh >=2.0.0*
+    - tethys_dataset_services >=2.0.0
+    - hs_restclient =1.3.*
+    - owslib =0.26.*
+    - requests =2.28.*
+    - social-auth-app-django =4.0.*
+    - dask =1.2.*
+    - tethys_dask_scheduler >=1.0.2
+    - channels =2.*
+    - daphne =2.5.*
+    - service_identity =18.1.*
+    - condorpy =0.6.*
+    - siphon =0.9
+    - python-jose =3.3.*
+    - pyjwt <2.0.0
+    - arrow =1.2.*
+    - isodate =0.6.*
+    - django =2.2.*
+    - django-axes =5.35.*
+    - django-filter =22.1
+    - djangorestframework =3.13.*
+    - django-bootstrap3 =11.1.*
+    - django-model-utils =4.0.*
+    - django-guardian =2.4.*
+    - django-gravatar2 =1.4.*
+    - django-termsandconditions =2.0.*
+    - django-session-security =2.6.*
+    - django-analytical =3.1.*
+    - django-simple-captcha =0.5.*
+    - django-recaptcha2 =1.4.*
+    - django-mfa2 <2.5
+    - social-auth-app-django =4.0.*
+    - requests-mock =1.9.*
+    - factory_boy =3.2.*
+    - pillow =9.2.*
+    - pip =22.1.*
+    - future =0.18.*
+    - pbr =5.9.*
+    - git
+    - conda
```

Comment:
Why are git and conda required at runtime?

Reply:
pip and conda are required because Tethys uses their APIs in the `tethys install` command to install dependencies of apps that it is installing.

Reply:
Removed git